experimental/telemetry/tracker.go

```diff
@@ -34,7 +64,9 @@ type tracker struct {
 	pluginID           string
 	pluginVersion      string
 	telemetryShortName string
-	enabled            bool
+	enabledTracking    bool
+	enabledLogging     bool
```

Comment:
1/5 should a `tracker` include a `TrackerConfig` instead?

Reply:
I don't feel the pressure yet, but I'm not against that if you perceive it as more readable/solid